src/common/args.h

```diff
@@ -420,6 +421,8 @@ class ArgsManager
      */
     const fs::path& GetDataDir(bool net_specific) const;
 
+    std::variant<ChainType, std::string> GetChainArg() const;
```

Comment:
In commit "refactor: Avoid needless chain type string conversions" (95744f9cf1f143b6449a5046a914557b3886e3a2)

Would be helpful to have a code comment describing what this does. Like "Return -regtest/-signet/-testnet/-chain= setting as a ChainType enum if a recognized chain name was set, or as a string if an unrecognized chain name was set. Raise an exception if an invalid combination of flags was provided.